packages/design-system-scripts/gulp/docs/runWebpackServer.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * The `server` task starts a Browsersync server to enable previewing the
+ * documentation site locally (localhost:3000). We integrate Browsersync with
+ * Webpack's hot reloading capability to enable tons of cool stuff like: form
+ * replication, scroll and click mirroring, and injecting updated CSS and JS
+ * without a page refresh :mindblown:
+ */
+
+const argv = require('yargs').argv;
+const path = require('path');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+const webpackHotMiddleware = require('webpack-hot-middleware');
+const webpackStatsConfig = require('../webpack-stats');
+
+module.exports = (gulp, shared) => {
```

Comment:
Note that this is a raw copy of the old script and hasn't been updated to move way from defining tasks with gulp. I mostly wanted the file names committed as a template of where I was going to go next